monkey/infection_monkey/monkey.py

```diff
@@ -129,6 +133,8 @@ def start(self):
         logger.info("Agent is starting...")
         logger.info(f"Agent GUID: {GUID}")
 
+        self._event_queue = PyPubSubEventQueue(Publisher())
```

Comment:
Why can't this be in the initializer?

Reply:
I may be convinced to that but I am not sure if the event queue is part of the `__init__` function. Same as the `monkey_inbound_tunnel`